scylla-cdc-base/src/test/java/com/scylladb/cdc/model/worker/RawChangeTest.java

```diff
@@ -111,7 +111,7 @@ public void testGetDeletedElements() {
 
         // v is not a nonfrozen collection, so it doesn't have
         // a cdc$deleted_elements_ column.
-        assertThrows(IllegalStateException.class, () -> insert1.getDeletedElements("v"));
+        assertThrows(IllegalArgumentException.class, () -> insert1.getDeletedElements("v"));
```

Comment:
Previously, the exception thrown was `IllegalStateException`: a `ColumnDefinition` could not return other `ColumnDefinition` (its internal state disallowed it). Now the exception is `IllegalArgumentException` - cannot `getDeletedElements` for provided argument `v`.

Ideally, we should maybe introduce our own exceptions (like `ColumnNotFoundException`, `CDCUnsupportedOperationException`), so that all methods return similar exceptions (not that one method returns `IllegalStateException` and other `IllegalArgumentException`). But this is outside of scope of this PR.